thread/src/components/RegisterForm.js

```diff
@@ -54,79 +78,98 @@ class RegisterForm extends Component {
         }
 
         return (
-            <Button full style={{ marginTop: 25 }} onPress={this.onSubmitPress}>
+            <Button rounded style={styles.buttonStyle} onPress={this.onSubmitPress}>
```

Comment:
I have made this button reusable in its own component, good idea to get my PRs reviewed as I have made changes to how we create reusable logic